### PR TITLE
Save branch

### PR DESCRIPTION
**Pull request by NaveenGoli**
Branding changes

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@NaveenGoli__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ngoli). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by AzureRT** [comment]
Can one of the admins verify this patch?

File: src/ResourceManager/Websites/Commands.Websites/Cmdlets/GetAzureWebApp.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@
 using Microsoft.Azure.Management.WebSites.Models;
 using Microsoft.WindowsAzure;
 using Microsoft.WindowsAzure.Commands.Utilities.CloudService;
-using Microsoft.Azure.Commands.Websites;
+using Microsoft.Azure.Commands.Webpp;
```
Comment (amitapl) [inline src/ResourceManager/Websites/Commands.Websites/Cmdlets/GetAzureWebApp.cs]: Webpp --> WebApp

File: src/ResourceManager/Websites/Commands.Websites/Microsoft.Azure.Commands.Websites.dll-Help.xml [hunk 2]
```diff
@@ -402,6 +414,145 @@
 <command:command xmlns:maml="http://schemas.microsoft.com/maml/2004/10" xmlns:command="http://schemas.microsoft.com/maml/dev/command/2004/10" xmlns:dev="http://schemas.microsoft.com/maml/dev/2004/10" xmlns:MSHelp="http://msdn.microsoft.com/mshelp">
 <!--Generated by PS Cmdlet Help Editor-->
 	<command:details>
+		<command:name>Get-AzureWebsitePublishingProfile</command:name>
```
Comment (amitapl) [inline src/ResourceManager/Websites/Commands.Websites/Microsoft.Azure.Commands.Websites.dll-Help.xml]: webapp?

File: src/ResourceManager/Websites/Commands.Websites/Microsoft.Azure.Commands.Websites.dll-Help.xml [hunk 3]
```diff
@@ -402,6 +414,145 @@
...
+	</maml:alertSet>
+	<command:examples>
+	</command:examples>
+	<maml:relatedLinks>
+	</maml:relatedLinks>
+</command:command>
+<command:command xmlns:maml="http://schemas.microsoft.com/maml/2004/10" xmlns:command="http://schemas.microsoft.com/maml/dev/command/2004/10" xmlns:dev="http://schemas.microsoft.com/maml/dev/2004/10" xmlns:MSHelp="http://msdn.microsoft.com/mshelp">
+<!--Generated by PS Cmdlet Help Editor-->
+	<command:details>
 		<command:name>New-AzureWebHostingPlan</command:name>
```
Comment (amitapl) [inline src/ResourceManager/Websites/Commands.Websites/Microsoft.Azure.Commands.Websites.dll-Help.xml]: appserviceplan?

File: src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseClient.cs [hunk 4]
```diff
@@ -18,35 +18,25 @@
...
-
-namespace Microsoft.Azure.Commands.Websites.Cmdlets.WebHostingPlan
+namespace Microsoft.Azure.Commands.Websites.Models.WebApp
 {
-    /// <summary>
-    /// this commandlet will let you delete an Azure Web Hosting Plan using ARM APIs
-    /// </summary>
-    [Cmdlet(VerbsCommon.Get, "AzureWebHostingPlan"), OutputType(typeof(WebHostingPlanGetResponse))]
-    public class GetWebHostingPlanCmdlet : WebHostingPlanBaseCmdlet
+    public abstract class WebAppBaseClient : AzurePSCmdlet
```
Comment (amitapl) [inline src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseClient.cs]: WebAppBaseClient --> WebAppBaseCmdlet

File: src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseCmdlet.cs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Websites.Models;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Websites.Models.WebApp;
+
+namespace Microsoft.Azure.Commands.Webpp
+{
+    public class WebAppBaseCmdlet : WebAppBaseClient
```
Comment (amitapl) [inline src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseCmdlet.cs]: WebAppBaseCmdlet --> WebAppBaseMandatoryCmdlet

File: src/ResourceManager/Websites/Commands.Websites/Utilities/WebsitesClient.cs [hunk 6]
```diff
@@ -168,5 +150,51 @@ public WebHostingPlanListResponse ListWebHostingPlan(string resourceGroupName)
             //proper return type need to be discussed
             return response;
         }
+
+        public WebHostingPlanGetHistoricalUsageMetricsResponse GetWebHostingPlanHistoricalUsageMetrics(string resourceGroupName, string webHostingPlanName, IList<string> metricNames,
```
Comment (amitapl) [inline src/ResourceManager/Websites/Commands.Websites/Utilities/WebsitesClient.cs]: GetAppServicePlan...

File: src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseClient.cs (hunk 4 above)
Comment (NaveenGoli) [inline src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseClient.cs]: name of WebAppBaseClient is actually right see the name of the cs file..

File: src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseCmdlet.cs (hunk 5 above)
Comment (NaveenGoli) [inline src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseCmdlet.cs]: this is correct see the name of the cs file. the class name is WebAppBaseCmdlet

Comment (amitapl) [inline src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseCmdlet.cs]: not saying its not, just that we need to rename it

File: src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseClient.cs (hunk 4 above)
Comment (amitapl) [inline src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseClient.cs]: it's a comment I had before, the name client is incorrect for this class so I think it should be renamed

**Comment by amitapl** [comment]
Looks good :shipit:

**Comment by matt-gibbs** [comment]
no need to fake a change, just tell Jenkins to 
re-test this please

**Comment by NaveenGoli** [comment]
I tried to resched, perhaps I don't have right? so i made a fake change

**Comment by azurecompliance4** [comment]
How did you try to start another build?
Jenkins is controlled by specific comments in the PR like the one I did just before.

**Pull request merged**